components/TextFields/tests/snapshot/MDCTextAreaOutlinedControllerBaselineCharacterCountArabicSnapshotTests.m

```diff
@@ -44,14 +43,9 @@ - (void)setUp {
   self.textFieldController.characterCountViewMode = UITextFieldViewModeAlways;
   MDCSemanticColorScheme *colorScheme =
       [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-  MDCTypographyScheme *typographyScheme =
-      [[MDCTypographyScheme alloc] initWithDefaults:MDCTypographySchemeDefaultsMaterial201804];
 
   [MDCOutlinedTextFieldColorThemer applySemanticColorScheme:colorScheme
```

Comment:
If there is no theming extension, please add a TODO to re-apply the theming once it's created.

Reply:
I personally don't think adding theming extensions for MDCTextInputControllerOutlinedTextArea is a worthwhile endeavor. I'm hoping that MDCTextControl text areas become the preferred solution before text areas before that work would be planned. What do you think about that?

Reply:
We may decide not to build Theming Extensions for the existing Text Fields, but having a TODO to track the work and know where to update will assist us if we do decide to build them.